0000-pre-shared-key-v2.md

```diff
@@ -234,6 +231,42 @@ The following is encrypted, as defined [above](#encryption). In the case of a re
 
 **TODO:** should the payment amount support numbers larger than UInt64s?
 
+#### Payment Chunk Response
+
+The following is encrypted, as defined [above](#encryption), and set as the data accompanying the payment chunk fulfillment.
+
+```
+(Numbers represent bytes)
+
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|T|           Payment ID          |  Seq  | Payment Rcved | Chu-|
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|nk Rcved |             Application Data                        |
```

Comment:
Maybe include the length prefix for the application data in this diagram

Reply:
I'd like to but I'm not sure how to illustrate a variable length prefix...